### ISSUE TITLE
The added submit button displays "Submit," but the assertion still fails

### ISSUE THREAD

**Issue by NicksPatties** [freeCodeCamp/freeCodeCamp#48536]
### Describe the Issue

This step of the Accessibility Quiz lesson asks the user to add a Submit button to their form to finish things off. My button says "Submit," but the assertion still fails and the lesson cannot be completed.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-41

### Your code

```html
<input type="submit"/>
```

### Expected behavior

I expected the test to pass, since my button says "Submit."

### Screenshots

The assertion message says the submit button should say "Submit..."
![Screen Shot 2022-11-17 at 3 32 18 PM](https://user-images.githubusercontent.com/3261001/202575298-eb49419d-0f56-4c94-b60b-5aaafe41dad5.png)

...but it does, leading to some confusion.
![Screen Shot 2022-11-17 at 3 32 32 PM](https://user-images.githubusercontent.com/3261001/202575312-7bd312e4-14fb-417f-9df0-e438a5cfe3cd.png)

### System

- Device: Laptop
- OS: macOS 10.15.7
- Browser: Brave
- Version: 1.45.127

### Additional context

Since what is actually displayed in the button can vary from browser to browser if there's no `value` property in the input, perhaps the hint can specifically state that the `value` be "Submit." So instead of
```
The submit should display the text "Submit"
```
perhaps this hint may work instead (or something similar):
```
The value of the submit input should be "Submit"
```

Note this also doesn't consider the case of using a `<button>` element to submit the form.

**Comment by naomi-lgbt** [comment]
Hi there,

Thanks for taking the time to open this issue. We're going to close this as a duplicate of #47447.
